reducers/queues.js

```diff
@@ -172,6 +173,18 @@ const queues = (state = defaultState, action) => {
     case DELETE_QUESTION.SUCCESS: {
       return removeQuestionFromQueue(state, action.queueId, action.questionId)
     }
+    case UPDATE_QUEUE.SUCCESS: {
+      const { queue } = action
+      // Manually preserve questionCount,
```

Comment:
Old comment?

Reply:
Oops...forget to remove that when fixing the API.